.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+version: 2.0
+
+jobs:
+  "Test":
+    docker: # Replace this with a fastlane owned docker image at some point
+      - image: joshdholtz/fastlane-docs:0.0.1
+    environment:
+      LC_ALL: C.UTF-8
+      LANG: C.UTF-8
+    working_directory: ~/code
+    steps:
+      - checkout
+      - run: pip3 install pipenv
```

Comment:
`pip` (circle.yml) vs. `pip3` - What the difference?

Reply:
`pip3` is for python 3. Guessing the newer 2.0 images have newer python? 🐍

Reply:
I made this newer image have python 3 so `pip3` is what we need here 😊